internal/fwserver/server_importresourcestate_test.go

```diff
@@ -299,6 +303,42 @@ func TestServerImportResourceState(t *testing.T) {
 				},
 			},
 		},
+		"response-importedresources-deferral": {
```

Comment:
Nit: It would be good to have unit testing cases here and for the other RPCs for request client capabilities -- both the zero-value and when enabled 👍